angular-workspace/package.json

```diff
@@ -22,24 +22,24 @@
     "format": "ng lint --fix"
   },
```

Comment:
Should also make a tech debt to see if we can simplify our directives by leveraging that `@Input({transform: someTransformMethod})` feature. Could potentially shrink the lines of code and delete most of the tests 🤞.

Reply:
Created #2010